src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedDayNames.cs

```diff
@@ -23,11 +23,11 @@ public void AbbreviatedDayNames_Set()
             Assert.Equal(newAbbreviatedDayNames, format.AbbreviatedDayNames);
         }
 
-        [Fact]
+        [Fact ActiveIssue(13247, TestPlatforms.AnyUnix)]
         public void AbbreviatedDayNames_Set_Invalid()
         {
             Assert.Throws<ArgumentNullException>("value", () => new DateTimeFormatInfo().AbbreviatedDayNames = null); // Value is null
-            Assert.Throws<ArgumentNullException>("value", () => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
+            Assert.Throws<ArgumentNullException>("values[3]", () => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
```

Comment:
You're updating this value and disabling the test?  I suggest keeping the ActiveIssue but reverting this change, such that whoever investigates the behavior change can investigate whether it was intentional or not, and only if it was update the test to match.